helix-core/src/test/java/org/apache/helix/integration/TestCustomizedViewAggregation.java

```diff
@@ -410,7 +410,9 @@ public void testCustomizedViewAggregation() throws Exception {
     // Aggregating: Type A
     // Routing table: Type A, Type B, Type C
     setAggregationEnabledTypes(Arrays.asList(CustomizedStateType.TYPE_A));
```

Comment:
If the case that reduce customized state config won't be tested at all, I suggest to comment out the whole case.

Reply:
The reduced config is tested, it is just not validated alone, it's validated together with the next several changes.

Reply:
ok, in that case just document a little more about how this test is performed.

Reply:
Sure. Just updated, please take a look :)